labs/b6.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Lab 6 - Processes and Services
+layout: lab
+---
+
+## Overview
+At any given moment, there are anywhere from dozens to thousands of process running on a Unix system. The large majority of these processes, called daemons, run in the background. Daemons are crucial to having a usable system and provide much of a system's core functionality, including the graphics server, sound server, and networking services to name a few.
+
+Today we'll explore these background processes and create some of our own! Make sure the following exercises are executed on your VM unless explicitly specified otherwise.
+
+## Which processes are running on my system?
+Open up a terminal and run the `ps` command. You should see something like this: 
+```
+  PID TTY          TIME CMD
+ 3634 pts/2    00:00:00 ps
+15015 pts/2    00:00:00 bash
+```
+
+Now open up **another** terminal and run `sleep 1000` in the background, and then run `ps`. It should look like:
+```
+$ sleep 1000 &
+[1] 29756
+$ ps
+  PID TTY          TIME CMD
+29446 pts/2    00:00:00 bash
+29756 pts/2    00:00:00 sleep
+29757 pts/2    00:00:00 ps
+```
+In the **first** terminal run `ps` again. You should notice that the `sleep` process is not showing up, even though the thousand seconds haven't expired. Why do you think this behavior occurs (hint: TTY column)?
+
+We can get the process to display on the first terminal by running `ps -u`, which displays all the processes running as your user. Notice the PID column; each process has a unique ID assigned to it by the kernel. One thing we can do with this PID is send signals to the process. `sleep 1000` is pretty useless, so go ahead and kill it --    `kill 29756` (substitute `29756`  with whatever `ps` outputted for you). 
```

Comment:
What's up with the extra spaces after the dash?

Also, I'd suggest putting in real em dashes or just using a single dash - two dashes looks somewhat ugly in sans serif